cmd/helm/root_test.go

```diff
@@ -17,8 +17,8 @@ limitations under the License.
 package main
 
 import (
+	"k8s.io/helm/pkg/helm/helmpath"
 	"os"
-	"path/filepath"
 	"testing"
 
 	"k8s.io/client-go/util/homedir"
```

Comment:
One more thing, the CI gate is failing because "k8s.io/client-go/util/homedir" is no longer used. You'll need to delete this line.

Also, thanks for the quick response and fix :smile:

Reply:
Fixed this one too :) Thank you :D